FrameworkSystem/DB/DynamicMonitoringDB.py

```diff
@@ -0,0 +1,182 @@
+"""
+Class and utilities for managing dynamic monitoring logs in Elasticsearch
+"""
+
+__RCSID__ = "$Id$"
+
+import datetime
+from DIRAC.Core.Utilities.ElasticSearchDB import ElasticSearchDB
+from DIRAC import gLogger, gConfig, S_OK, S_ERROR
+
+class DynamicMonitoringDB( object ):
```

Comment:
Usual pattern in DIRAC is that DB classes inherit from "DB" (which in this case would be "ElasticSearchDB"). Does it need to be different here?